queries/ruby/highlights.scm

```diff
@@ -88,7 +88,7 @@
 (keyword_parameter (identifier) @parameter)
 
 ((identifier) @function
- (is-not? local))
+ (#is-not? local))
```

Comment:
I just see this predicate, and it is not supported (yet)

Reply:
Do we want to remove it for now?

Reply:
Yes maybe, more like commenting it as a TODO for me for example